### ISSUE TITLE
When typing in Chinese input method, the program will automatically close

### ISSUE THREAD

**Issue by jadeyx** [zed-industries/zed#15185]
### Check for existing issues

- [ ] Completed

### Describe the bug / provide steps to reproduce it

error: process didn't exit successfully: `target\release\zed.exe` (exit code: 0xffffffff)

### Environment

OS: windows 11

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by notpeter** [comment]
This might be a duplicate of:
- https://github.com/zed-industries/zed/issues/12563

**Comment by JunkuiZhang** [comment]
Which input method are you using? It's ok on my machine, both win10 and win11.


https://github.com/user-attachments/assets/8accc325-fe3d-4912-9321-c00151451993

**Comment by jadeyx** [comment]
> 你用的是什么输入法？我的机器上没问题，win10 和 win11 都可以。
> 
>  录音.2024-07-26.001013.mp4

Default Microsoft Pinyin input method


https://github.com/user-attachments/assets/9d7ab268-0f34-4a92-936c-ff37ba56d39e

**Comment by JunkuiZhang** [comment]
没有日志输出吗？你能先设置$env:RUST_LOG=“INFO”，然后再cargo run吗？

**Comment by jadeyx** [comment]
> 没有日志输出吗？你能先设置$env:RUST_LOG=“INFO”，然后再cargo run吗？

```bash
Thread "main" panicked with "called `Option::unwrap()` on a `None` value" at crates\gpui\src\platform\windows\platform.rs:157:14
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: git_smart_subtransport_git
  10: git_smart_subtransport_git
  11: git_smart_subtransport_git
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: <unknown>
  22: git_smart_subtransport_git
  23: BaseThreadInitThunk
  24: RtlUserThreadStart
```

**Comment by JunkuiZhang** [comment]
https://github.com/zed-industries/zed/blob/243fb3562c79b233389cea035a660bff6af857ff/crates/gpui/src/platform/windows/events.rs#L19

你把这行改成`pub(crate) const CLOSE_ONE_WINDOW: u32 = WM_USER + 200; `试试，还会崩溃吗？

**Comment by shenjackyuanjie** [comment]
> https://github.com/zed-industries/zed/blob/243fb3562c79b233389cea035a660bff6af857ff/crates/gpui/src/platform/windows/events.rs#L19
> 
> 你把这行改成`pub(crate) const CLOSE_ONE_WINDOW: u32 = WM_USER + 200; `试试，还会崩溃吗？

我试试去

**Comment by notpeter** [comment]
Is this still an issue or was it fixed by:
- https://github.com/zed-industries/zed/pull/15286

**Comment by jadeyx** [comment]
> Is this still an issue or was it fixed by:
> 
> * [windows: Fix some weird IME window panic #15286](https://github.com/zed-industries/zed/pull/15286)

Has been fixed
